research/src/pages/checkbox.proposal.mdx

```diff
@@ -0,0 +1,82 @@
+---
+menu: Components
+name: Checkbox (Working Draft)
+path: /components/checkbox
+pathToResearch: /components/checkbox.research
+---
+
+import CheckboxAnatomy from '../components/checkbox-anatomy'
+
+The `<checkbox>` element is a control that allows the user to select a binary state, either checked or unchecked.
```

Comment:
probably worth denoting indeterminate state

Reply:
Good catch - it's missing from the states / properties section also. I'll add.